### ISSUE TITLE
some questions for the 3D average precision (AP) results on the STF

### ISSUE THREAD

**Issue by RhythmOfTheRain-Byte** [MartinHahner/LiDAR_fog_sim#15]
Hello, I have read your paper of "Fog Simulation on Real LiDAR Point Clouds for 3D Object Detection in Adverse Weather," and I have some questions about it. Specifically, I used the STF dataset and divided the clear data into training, testing, and validation sets in a 7:2:1 ratio. I trained the PointPillar and PV-RCNN++ models on the clear training dataset and tested them on the clear validation set based on openpcdet. However, the results are not satisfactory, and I am unable to achieve the impressive results mentioned in your article. This issue has been bothering me for a long time. I apologize for the inconvenience, and I seek your assistance. Thank you.

some results as fellows:
Pedestrian AP@0.50, 0.50, 0.50:
bbox AP:15.5895, 15.2841, 15.5391
bev  AP:11.0431, 11.1163, 11.3700
3d   AP:10.4871, 10.5488, 10.2467
aos  AP:11.37, 11.10, 11.26
Pedestrian AP_R40@0.50, 0.50, 0.50:
bbox AP:9.3038, 9.4138, 9.3635
bev  AP:3.7221, 3.5240, 3.7916
3d   AP:2.8358, 2.6212, 2.5333
aos  AP:5.32, 5.35, 5.32
Pedestrian AP@0.50, 0.25, 0.25:
bbox AP:15.5895, 15.2841, 15.5391
bev  AP:15.1044, 15.0117, 15.5118
3d   AP:14.8881, 14.7912, 15.3198
aos  AP:11.37, 11.10, 11.26
Pedestrian AP_R40@0.50, 0.25, 0.25:
bbox AP:9.3038, 9.4138, 9.3635
bev  AP:8.7162, 8.9601, 9.1860
3d   AP:8.4221, 8.4356, 8.8460
aos  AP:5.32, 5.35, 5.32
PassengerCar AP@0.70, 0.70, 0.70:
bbox AP:0.0000, 0.0000, 0.0000
bev  AP:0.0000, 0.0000, 0.0000
3d   AP:0.0000, 0.0000, 0.0000
aos  AP:0.00, 0.00, 0.00
PassengerCar AP_R40@0.70, 0.70, 0.70:
bbox AP:0.0000, 0.0000, 0.0000
bev  AP:0.0000, 0.0000, 0.0000
3d   AP:0.0000, 0.0000, 0.0000
aos  AP:0.00, 0.00, 0.00
PassengerCar AP@0.70, 0.50, 0.50:
bbox AP:0.0000, 0.0000, 0.0000
bev  AP:0.0000, 0.0000, 0.0000
3d   AP:0.0000, 0.0000, 0.0000
aos  AP:0.00, 0.00, 0.00
PassengerCar AP_R40@0.70, 0.50, 0.50:
bbox AP:0.0000, 0.0000, 0.0000
bev  AP:0.0000, 0.0000, 0.0000
3d   AP:0.0000, 0.0000, 0.0000
aos  AP:0.00, 0.00, 0.00
RidableVehicle AP@0.50, 0.50, 0.50:
bbox AP:0.0000, 0.0000, 0.0000
bev  AP:0.0000, 0.0000, 0.0000
3d   AP:0.0000, 0.0000, 0.0000
aos  AP:0.00, 0.00, 0.00
RidableVehicle AP_R40@0.50, 0.50, 0.50:
bbox AP:0.0000, 0.0000, 0.0000
bev  AP:0.0000, 0.0000, 0.0000
3d   AP:0.0000, 0.0000, 0.0000
aos  AP:0.00, 0.00, 0.00
RidableVehicle AP@0.50, 0.25, 0.25:
bbox AP:0.0000, 0.0000, 0.0000
bev  AP:0.0000, 0.0000, 0.0000
3d   AP:0.0000, 0.0000, 0.0000
aos  AP:0.00, 0.00, 0.00
RidableVehicle AP_R40@0.50, 0.25, 0.25:
bbox AP:0.0000, 0.0000, 0.0000
bev  AP:0.0000, 0.0000, 0.0000
3d   AP:0.0000, 0.0000, 0.0000
aos  AP:0.00, 0.00, 0.00
